### DISCUSSION TITLE
After downloading the latest update for the iOS app, my enterprise profile is totally broken. Only on GitHub mobile.

### DISCUSSION THREAD

**Discussion by kmoltke** [community/community#48426] [Enterprise]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.100.0 (91333308)
Device: iPhone 12 Pro (iOS 16.3.1)
</details>

**Comment by candyho** [comment]
Hey, sorry for the hassle. It'd be a huge help if you could share a screenshot with us to see what's broken to help us debug. Also, are you logging in with a Enterprise Server account or a normal Enterprise Cloud account?

**Reply by kmoltke** [reply]
![image](https://user-images.githubusercontent.com/38263457/221623292-f3df3e66-12a9-408c-8ee3-28707f573094.png)

**Reply by kmoltke** [reply]
I have no idea what the difference between server and cloud enterprise is. The domain is itu.dk

**Comment by weom** [comment]
Also facing the same error, so just dropping some screenshots of the error messages to see if it will be helpful. 🙇‍♂️

GitHub for iOS, Version 1.100.0 (91333308) Device: iPhone 13 (iOS 16.3.1)

1st Error:
<img src="https://user-images.githubusercontent.com/10952871/221503267-297892a4-a6e8-4eea-833f-dd0832381cb6.png" width="300">

2nd Error:
<img src="https://user-images.githubusercontent.com/10952871/221503278-411e37ea-bf86-4105-aa6d-7abaab89c597.png" width="300">
